src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.vb

```diff
@@ -321,46 +321,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return True
         End Function
 
-        Private Shared Function HasNonDefaultTupleElementNames(tupleSymbol As INamedTypeSymbol) As Boolean
-            Dim elementNames = tupleSymbol.TupleElementNames
-
-            If Not elementNames.IsDefault Then
-                For i As Integer = 0 To elementNames.Length - 1
-                    If (elementNames(i) <> TupleTypeSymbol.TupleMemberName(i + 1)) Then
-                        Return True
-                    End If
-                Next
-            End If
-
-            Return False
+        Private Shared Function HasNonDefaultTupleElements(tupleSymbol As INamedTypeSymbol) As Boolean
+            Return tupleSymbol.TupleElements.Any(Function(e) e IsNot e.CorrespondingTupleField)
```

Comment:
Yes, you might want "IsDefault" and "HasUserProvidedTupleName" and possibly "UserProvidedNameOrNull"

Note they are subtly different, since user can provide "Item1", then it will be user-provided and yet default.

I was not sure where such extensions would go.